test/ui-testing/amendments.js

```diff
@@ -92,11 +92,15 @@ module.exports.test = (uiTestCtx) => {
 
             amendment.terms.forEach(term => {
               chain = chain
+                .wait('#add-term-btn')
                 .click('#add-term-btn')
+                .wait(500)
                 .evaluate(() => [...document.querySelectorAll('[data-test-term]')].length - 1)
                 .then(index => {
                   nightmare
-                    .type(`#edit-term-${index}-name`, term.label)
+                    .wait(`#edit-term-${index}-name`)
+                    .type(`#edit-term-${index}-name`, term.input)
```

Comment:
What was the issue here? How does this solve it?

Reply:
This is really weird. I realized that typing 'Other restrictions' always ended up with choosing the wrong term - the only one beginning with the **second** letter 'T':`Text and Data Mining`. As a result, an error message (because the terms value couldn't be set) prevented the amendment's being saved. So this is just a workaround because I have no idea how to solve this. Obviously it's only working as long as there's just one additional term beginning with the letter 'O'.

Reply:
Odd! And my first guess about what caused this should've been handled by the .wait(500) you added...